mne_bids/write.py

```diff
@@ -2543,18 +2551,24 @@ def mark_channels(bids_path, *, ch_names, status, descriptions=None, verbose=Non
             f"({len(ch_names)})."
         )
 
-    if not all(status in ["good", "bad"] for status in status):
+    if set(status) != {"good", "bad"}:
```

Comment:
I think this is what might be needed:

```suggestion
    if not set(status).issubset({"good", "bad"}):
```